### PR TITLE
Fix: profile left alignment

### LINKED ISSUE #88: Minor: misaligned title in user home, in viewport interval 656px up to 767px
In the user homepage, between 656px and 767px wide, the profile summary title is left aligned.
It should be the same as the title above it (quick actions).

image

### PR DESCRIPTION
**Pull request by basilesimon**
Hopefully address #88 

https://user-images.githubusercontent.com/5701152/153442568-60b6f1b3-ce67-4a8f-84b1-2256d4126144.mov

### CODE REVIEW

File: src/modules/home/HomePage.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
       <font-awesome-icon class="transform -rotate-90" :icon="['fas', 'key']" />
     </template>
 
-    {{ t('homePage.passwordReset') }}
+    {{ `${t('homePage.passwordReset')}` }}
```
Comment (wpf500) [inline src/modules/home/HomePage.vue]: I don't think this is needed?

**Review by wpf500 — CHANGES_REQUESTED** [review]
Looks good apart from what looks like an accidental commit :+1:, can you revert that line?

File: src/modules/home/HomePage.vue (hunk 1 above)
Comment (basilesimon) [reply src/modules/home/HomePage.vue]: Mmmh, funny things are happening with the linter/fixer triggering upon committing. I managed to revert it – will let you mark this as resolved.

**Review by wpf500 — APPROVED** [review]

File: src/modules/home/HomePage.vue (hunk 1 above)
Comment (wpf500) [reply src/modules/home/HomePage.vue]: Yeah we should definitely review whether the linter/fixer triggering on commit is useful or not, I find it a bit jarring

**Pull request merged**